app/brave_generated_resources.grd

```diff
@@ -250,6 +250,11 @@
           Restart browser to enable Google Widevine
         </message>
       </if>
+      <if expr="is_android">
+        <message name="IDS_WIDEVINE_PERMISSION_REQUEST_TEXT_FRAGMENT_ENABLE_SYSTEM" desc="Text fragment for Widevine permission request. 'Widevine' is the name of a plugin and should not be translated.">
+          <ph name="URL">$1<ex>https://www.youtube.com</ex></ph> wants to play protected Google Widevine content, which requires you to allow Android system support for Widevine. By allowing, you'll agree to Google's terms of use.
```

Comment:
Suggested change:

... wants to play content protected by DRM. Playing this content requires that you allow system support for Google Widevine on your Android. By clicking "Allow", you are agreeing to Google Widevine's terms of use.

Assuming there's an "Allow" button in the dialog.  cc: @rmcfadden3, do you have any suggestions for this text?

Reply:
@mkarolin — I believe I already helped review / edit this text, in a Slack thread with Time.

@timchilds can you comment?

Reply:
Ok, thanks. Approving then.